app/controllers/tags_controller.rb

```diff
@@ -2,7 +2,8 @@ class TagsController < ApplicationController
   before_filter :load_collection
   before_filter :check_user_status, :except => [ :show, :index, :show_hidden, :search, :feed ]
   before_filter :check_permission_to_wrangle, :except => [ :show, :index, :show_hidden, :search, :feed ]
-  before_filter :load_tag, :only => [:show, :edit, :update, :wrangle, :mass_update]
+  before_filter :load_tag, :only => [:edit, :update, :wrangle, :mass_update]
+  before_filter :load_tag_and_subtags, :only => [:show]
```

Comment:
One of the responses on http://stackoverflow.com/questions/17807475/before-action-in-user-controller-rendering-error-nomethoderror-in-usersshow is indeed that `before_action` is only available in Rails 4.x and we're on 3.x, so we'll have to ignore that (there's some way of disabling specific Hound checks though I haven't tried it myself)